.github/workflows/base-component-descriptor.yaml

```diff
@@ -26,6 +26,23 @@ on:
         required: false
         type: string
         default: SAP SE
+      component-references:
```

Comment:
naming: should we rather call it component-references-path (or: -file)? I know: will lead to more typing efforts + if checking a concrete example (with a value), it should be obvious it is a filepath, so it is also fine w/ me to not add this suffix.

OTOH, we _might_ see cases / the need to pass-in crefs (from another action... - likely not as a value from calling the workflow). So adding a `-file` (or `-path`) suffix will leave us this name open